monai/apps/deepgrow/transforms.py

```diff
@@ -819,6 +819,10 @@ def __init__(
         original_shape_key: str = "foreground_original_shape",
         cropped_shape_key: str = "foreground_cropped_shape",
         allow_missing_keys: bool = False,
+        restore_resize: bool = True,
+        restore_crop: bool = True,
+        restore_spacing: bool = True,
+        restore_slicing: bool = True,
```

Comment:
These look fine to me as additions, but please add documentation in the docstring describing what they do and their defaults.